jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -44,19 +34,29 @@
       ></v-text-field>
     </v-row>
 
-    <v-row justify="end">
-      <j-tooltip tipid='plugin-moment-maps-calculate'>
-        <v-btn color="primary" text @click="calculate_moment">Calculate</v-btn>
-      </j-tooltip>
-    </v-row>
+    <plugin-add-results
+      :label.sync="results_label"
+      :label_default="results_label_default"
+      :label_auto.sync="results_label_auto"
+      :label_invalid_msg="results_label_invalid_msg"
+      :label_overwrite="results_label_overwrite"
+      label_hint="Label for the collapsed cube"
+      :add_to_viewer_items="add_to_viewer_items"
+      :add_to_viewer_selected.sync="add_to_viewer_selected"
+      action_label="Calculate"
+      action_tooltip="Calculate moment map"
+      @action-click="calculate_moment"
+    ></plugin-add-results>
 
     <div v-if="moment_available">
       <j-plugin-section-header>Results</j-plugin-section-header>
       <v-row>
           <v-text-field
            v-model="filename"
            label="Filename"
-           :rules="[() => !!filename || 'This field is required']">
+           hint="Export the latest calculated moment map"
```

Comment:
added this to clarify that the moment map isn't recomputed, but rather the latest one that was calculated is exported.  We may want to change this behavior in the future, but that is out of scope here.